pages/opensource.js

```diff
@@ -105,38 +105,15 @@ const Page = ({ repos }) => (
         All open sourced through Hack Club Bank Transparency Mode.
       </Text>
       <Grid columns={2} gap={3} mt={2} mb={[4]}>
-        <BankProject name="HQ" url={`https://bank.hackclub.com/hq/`} />
-        <BankProject
-          name="Endowment"
-          url={`https://bank.hackclub.com/endowment`}
-        />
-        <BankProject
-          name="Discretionary Fund"
-          url={`https://bank.hackclub.com/discretionary-fund`}
-        />
-        <BankProject
-          name="Summer of Making"
-          desc={`Our 2020 Summer Program`}
-          url="https://bank.hackclub.com/summer-of-making"
-        />
-        <BankProject
-          name="Summer of Making Stickers"
-          url="https://bank.hackclub.com/som-sticker-shipments"
-        />
-        <BankProject
-          name="The Hacker Zephyr"
-          url={`https://bank.hackclub.com/zephyr`}
-        />
-        <BankProject
-          name="Community Team"
-          url="https://bank.hackclub.com/community-team"
-        />
-        <BankProject
-          name="Wild Wild West"
-          url="https://bank.hackclub.com/wild-wild-west"
-        />
-        <BankProject name="Assemble" url="https://bank.hackclub.com/assemble" />
-        <BankProject name="Epoch" url="https://bank.hackclub.com/epoch" />
+        {transparentAccounts
+          .filter(account => account.category === 'hack_club_hq')
```

Comment:
```suggestion
          .filter(account => account.category === 'hack club hq')
```

I made this change earlier but it seems to have been reverted, the page doesn't work without the spaces left in.

Reply:
Oops I think this is actually a Bank bug; the value for the category field is supposed to be in snake case.

Reply:
Interesting, let's leave it as is for now I guess and change it later down the line, it's working great!